zanata-war/src/main/webapp/account/inactive_account.xhtml

```diff
@@ -12,6 +12,22 @@
     <a4j:jsFunction name="sendActivationEmail"
       action="#{inactiveAccountAction.sendActivationEmail}"/>
 
+    <script type="application/javascript">
+      function getNewEmailField() {
+        return jQuery('#' + jqSelector('#{rich:clientId('emailInput')}'));
+      }
+
+      jQuery(document).keypress(function(e) {
```

Comment:
Lets make the event listener be on a narrower object, like the form, instead of the whole document.